src/Java.Interop/Java.Interop/JniRuntime.JniMarshalMemberBuilder.cs

```diff
@@ -27,6 +28,11 @@ partial class CreationOptions {
 		}
 
 		[System.Diagnostics.CodeAnalysis.SuppressMessage ("Design", "CA1031:Do not catch general exception types", Justification = "the *.Export assemblies are optional, so we don't care when they cannot be loaded (they are presumably missing)")]
+#if NET
```

Comment:
"Philosophical" question here: *does it work*?

The warning mentioned:

    external\Java.Interop\src\Java.Interop\Java.Interop\JniRuntime.JniMarshalMemberBuilder.cs(53,4): warning IL2026: Java.Interop.JniRuntime.SetMarshalMemberBuilder(JniRuntime.CreationOptions): Using method 'System.Reflection.Assembly.GetType(String)' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. Types might be removed.

is *probably* coming from the `src/Mono.Android` build? https://github.com/xamarin/xamarin-android/blob/02c33474396f2f7dcd0dfadc25b0abb9f1b2d383/src/Mono.Android/Mono.Android.targets#L15-L27

which uses https://github.com/xamarin/java.interop/blob/8898bc1402953fdf3a1e1066dd2542fc1818aadf/src/Java.Interop/Java.Interop-MonoAndroid.csproj

which probably *doesn't* have `NET` defined.

Reply:
I tested it and the warning went away.

I think this works, because:

https://github.com/xamarin/xamarin-android/blob/02c33474396f2f7dcd0dfadc25b0abb9f1b2d383/src/Mono.Android/Mono.Android.csproj#L96-L97

When `TargetFramework` is `net6.0` in `Mono.Android.csproj`, it has a regular `<ProjectReference/>` for `Java.Interop.csproj` which should also be `net6.0`.